### PR TITLE
docs(suite): Update commit hook for fixup commits

### PR DESCRIPTION
**Pull request by jvaclavik**
## Description

The previous hook was not supporting fixup commits.

### CODE REVIEW

File: COMMITS.md [hunk 1]
```diff
@@ -16,10 +16,18 @@ Use this git hook to auto-check your commit messages. Save the following snippet
...
 ```
 #!/bin/sh
+
+commit_msg=$(cat "$1")
+if echo "$commit_msg" | grep -qE "^fixup! "; then
+  # Skip validation in case of fixup commit
+  exit 0
+fi
+
 if ! grep -qE "^(build|ci|docs|feat|fix|perf|refactor|style|test|chore|revert)(\((blockchain-link|components|components-storybook|integration-tests|suite|suite-data|suite-desktop|suite-native|suite-onboarding|suite-storage|suite-web)\))?: " "$1" ; then
```
Comment (mroz22) [inline COMMITS.md]: uh, this seems to be wildly outdated as well

Comment (jvaclavik) [reply COMMITS.md]: So what are the current rules?

Comment (mroz22) [reply COMMITS.md]: just that there are way more packages than this. also having this hardcoded is hard to mantain. I wouldn't mind simply removing pakage names rules from here

Comment (matejkriz) [reply COMMITS.md]: we can delete the recommendation of the hook completely, or at least remove the scope part check...

File: COMMITS.md [hunk 2]
```diff
@@ -16,10 +16,18 @@ Use this git hook to auto-check your commit messages. Save the following snippet
 
 ```
 #!/bin/sh
+
+commit_msg=$(cat "$1")
+if echo "$commit_msg" | grep -qE "^fixup! "; then
```
Comment (matejkriz) [inline COMMITS.md]: it would fail on Revert commits too, but those are not so common as fixups

File: COMMITS.md (hunk 1 above)
Comment (matejkriz) [inline COMMITS.md]: something like this?
```suggestion
if ! grep -qE "^(build|ci|docs|feat|fix|perf|refactor|style|test|chore|revert)(\([a-z]+\))?: " "$1" ; then
```
to allow 
`feat: anything`
`feat(foo): bar`

but not allow
`bla bal`
`feat bla`
`feat(232NOOO): bla`
`feat(a)`
...

File: COMMITS.md (hunk 2 above)
Comment (jvaclavik) [reply COMMITS.md]: Should be fixed now

File: COMMITS.md (hunk 1 above)
Comment (jvaclavik) [reply COMMITS.md]: Sounds good, I just added `-` character

Comment (matejkriz) [reply COMMITS.md]: cool, it works! 
image
image

**Review by matejkriz — APPROVED** [review]

**Pull request merged**